documentation/client_demo.ipynb

```diff
@@ -38,27 +56,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Used to one-time signup on model bazaar. Sends a email verification link.\n",
+    "# Sends a email verification link.\n",
     "bazaar.sign_up(email=\"your-mail-id@domain.com\", password=\"password\", username=\"username\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Login"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Login with your email_id and password.\n",
     "bazaar.log_in(email=\"your-mail-id@domain.com\", password=\"password\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creates a model from scratch on ModelBazaar -- trained on given list of documents."
+   ]
+  },
```

Comment:
@kartikTAI When are we planning to add `sharded_training` functionality?

Reply:
Planning to add it after https://github.com/ThirdAILabs/neuraldb-enterprise-services/pull/44 is merged. That PR will allow us to have versioning of the docker images, so we'll be able to create new releases easily.